x-pack/plugins/maps/public/classes/sources/es_source/es_source.ts

```diff
@@ -226,7 +226,17 @@ export class AbstractESSource extends AbstractVectorSource implements IESSource
 
       allFilters.push(extentFilter);
     }
-    if (searchFilters.applyGlobalTime && (await this.isTimeAware())) {
+
+    let isFeatureEditorOpenForLayer = false;
```

Comment:
isFeatureEditorOpenForLayer is no longer in searchFilters since its remove from dataFilters. You will need to update https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/classes/layers/build_vector_request_meta.ts#L14 to add it there so it gets passed into this method.